scala-generator/src/main/scala/models/Play2Json.scala

```diff
@@ -77,9 +77,19 @@ case class Play2Json(
   }
 
   private[models] def enumReadersAndWriters(enum: ScalaEnum): String = {
-    val jsObjectWriterMethod = play2JsonCommon.toJsonObjectMethodName(ssd.namespaces, enum.name)
     val jsValueWriterMethod = play2JsonCommon.implicitWriterName(enum.name)
-    val implicitWriter = play2JsonCommon.implicitWriter(enum.name, enum.qualifiedName, jsValueWriterMethod)
+    val jsObjectWriterMethod = play2JsonCommon.toJsonObjectMethodName(ssd.namespaces, enum.name)
+
+    val discriminator = getDiscriminator(enum)
+    val jsObjectBody = wrapInObject("play.api.libs.json.JsString(obj.toString)", discriminator)
```

Comment:
can you just inline this variable?

Reply:
Given there is some logic, I'd rather keep it out of the string interpolation.

What is the motivation?